src/elements/common/annotator-context/withAnnotatorContext.tsx

```diff
@@ -22,6 +31,7 @@ export default function withAnnotatorContext<P extends {}>(WrappedComponent: Rea
                     emitRemoveEvent={emitRemoveEvent}
                     getAnnotationsMatchPath={getAnnotationsMatchPath}
                     getAnnotationsPath={getAnnotationsPath}
+                    onAnnotationsFetchError={onAnnotationsFetchError}
```

Comment:
Does each context property need to be destructured individually or can we use a spread operator here?